js/tests/browsers.js

```diff
@@ -52,25 +52,25 @@ const browsers = {
         platformName: 'Windows 10'
       }
     },
-    iphone7: {
+    iphone11: {
```

Comment:
The reason we tested this version is because iOS 12 is supported. :/

Reply:
Let's try reverting it in that case, I just wanted to use devices which has high number of availability so that we can avoid timeout errors in most of the cases